SU-HSDO/suhumsci
# Arch 124 people views

# READY FOR REVIEW

# Summary
- People view with contextual filters.

# Needed By (Date)
- as soon as you can so Richard can style.

# Urgency
- Low

# Steps to Test

1. checkout this branch.
2. composer install/update
3. sync to production `blt drupal:sync --site=archaeology`
4. Use devel_generate or manually create some research areas first, then some people.
5. person affiliation terms should be automatically created
6. create/edit a basic page and add the Person views. use arguments to get a specific list of people. ie `Faculty` or `Staff+Undergraduate Students`
7. validate it has a similar look/feel to [this mockup](https://projects.invisionapp.com/share/GMJJYT1NS79#/screens/302442148) or [this mockup](https://projects.invisionapp.com/share/GMJJYT1NS79#/screens/302442149)
8. go to the entityqueue on `/admin/structure/entityqueue/hs_person/hs_person` page. Add a few pieces of content to the sort and sort them as desired.
9. verify that the sort you defined in the entityqueue is visible in the view. (sorted items will appear first in their defined sort. all other items will appear below them)


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
